### PR TITLE
Refactoring gulpfile.js.

### PR DESCRIPTION
**Pull request by will518**
- Replacing deprecated gulp-util package with fancy-log and ansi-colors dependencies directly.
- Refactoring the gulpfile.js to remove duplication.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,68 +1,75 @@
...
-  );
-  return serverStartPromise.then(() => lintHtml({
-    url: `http://localhost:${port}`,
-  }))
-    .then(() => process.exit(0))
-    .catch(e => gutil.log(gutil.colors.red(e)) && process.exit(1));
+  const url = `http://localhost:${port}`;
+
+  return new Promise(accept =>
+    http.createServer(app).listen(port, () => accept()))
```
Comment (chilliwilly77) [inline gulpfile.js]: check indentation. It might look fine in the editor to be honest.

**Comment by chilliwilly77** [comment]
👍 Can you make the same change to all the apps...? And the hello-world project 🚁

**Comment by will518** [comment]
@chilliwilly77 Are you able to quickly check that the last commit hasn't broken the html-lint functionality?

**Comment by will518** [comment]
@chilliwilly77 I've made a minor change after checking html-lint functionality.

**Pull request closed without merging**